crates/chain/src/tx_graph.rs

```diff
@@ -349,6 +349,11 @@ impl<A> TxGraph<A> {
             .filter(move |(_, conflicting_txid)| *conflicting_txid != txid)
     }
 
+    /// Iterates over all transaction anchors known by [`TxGraph`].
+    pub fn all_anchors(&self) -> impl ExactSizeIterator<Item = &(A, Txid)> + DoubleEndedIterator {
```

Comment:
why not just return the `&BTreeSet<..>`. Much more useful in a `.anchors` method.